cmd/tink-controller-v1alpha2/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.15
```

Comment:
any reason not to use `scratch`?

Reply:
I don't know. The goal of the PR is to get something functional so I've not experimented.